<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>750</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Data-Driven Software Security Assessment</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="file_select_btn">
      <property name="text">
       <string>Choose a File or Directory</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="text_browser">
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Times New Roman';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="menu_action_help"/>
    <addaction name="menu_action_add_api_key"/>
    <addaction name="menu_action_quit"/>
   </widget>
   <widget class="QMenu" name="menuExport">
    <property name="title">
     <string>Export</string>
    </property>
    <addaction name="menu_action_export_HTML"/>
    <addaction name="menu_action_export_PDF"/>
   </widget>
   <addaction name="menuOptions"/>
   <addaction name="menuExport"/>
  </widget>
  <action name="menu_action_help">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="menu_action_quit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit the program</string>
   </property>
  </action>
  <action name="menu_action_export_HTML">
   <property name="text">
    <string>HTML</string>
   </property>
  </action>
  <action name="menu_action_export_PDF">
   <property name="text">
    <string>PDF</string>
   </property>
  </action>
  <action name="menu_action_add_api_key">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Add API Key</string>
   </property>
   <property name="toolTip">
    <string>Add your NIST NVD API Key</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
